openwisp_notifications/api/serializers.py

```diff
@@ -59,6 +59,7 @@ class Meta(NotificationSerializer.Meta):
         fields = [
             'id',
             'message',
+            'get_description',
```

Comment:
Is the naming like this fine here? Would like to get some suggestions on how it can be.